### DISCUSSION TITLE
Not versed at github at all. how on earth do i update superslicer?

### DISCUSSION THREAD

**Discussion by mberzse1** [supermerill/SuperSlicer#4409] [Q&A] [answered]
I somehow updated superslicer once before. but now there's no information on how to do it again. Can someone please help me out?

**Comment by tmuehlhoff** [comment]
What did you saveare your Operating System details ?

**Reply by mberzse1** [reply]
Sorry. Didn't quite understand what your asking. If you're asking what OS details i have then Im running windows 11 64bit.

**Comment by Will-wastelander** [comment, marked as answer]
There is no "upgrade" for windows. You just download the new release .zip
extract it and run it instead of your old version.

**Reply by mberzse1** [reply]
Thank you! Man. I wish it would just say that somewhere.
